spring-cloud-gcp-starters/spring-cloud-gcp-starter-trace/pom.xml

```diff
@@ -5,6 +5,8 @@
 	<parent>
 		<artifactId>spring-cloud-gcp-starters</artifactId>
 		<groupId>org.springframework.cloud</groupId>
+
+		<!-- TODO: What version do I put here? -->
```

Comment:
Is it really a concern of a purpose of this PR?

This is `parent` version BTW, we can't live without it anyway...

Reply:
@dzou What's the concern around using the current version `1.1.0.BUILD-SNAPSHOT` as is done one all other starters?

Reply:
nevermind, I got confused originally; thought I had to increment the version number to accommodate a new starter.